README.md

```diff
@@ -36,6 +36,43 @@ Type the name of [any `iron-*` or `paper-*` element](https://elements.polymer-pr
 </dom-module>
 ```
 
+### [pe2015] polymer element ES2015
+
+```html
+<dom-module id="${2}">
+  <template>
+    <style>
+      :host {
+        display: block;
+      }
+    </style>
+
+  </template>
+  <script>
+    (function (Polymer) {
+      'use strict';
+      class ${1} {
+        beforeRegister() {
+          this.is = '${2}';
+
+          this.properties = {
+            ${3}
+          };
+        }
+
+        attached() {
+           this.async(() => {
```

Comment:
`ready` mean's the element's local dom children are upgraded, and you can start doing things like setting up properties. If you need to mess with light dom children you may want to either wait a tick after `attached` (using async as you have here) or use something like `observeNodes` in the case where you care about if light dom children are being dynamically added/removed